apps/site/tailwind.config.ts

```diff
@@ -1,161 +1 @@
-        500: '#AF74E8',
-        600: '#9756D6',
-        700: '#7D3CBE',
-        800: '#642B9E',
-        900: '#361B52',
-      },
-      accent2: {
-        100: '#FBF0F4',
-        200: '#FBD4E6',
-        300: '#FBB4D2',
-        400: '#F68BB7',
-        500: '#ED5393',
-        600: '#D6246E',
-        700: '#B01356',
-        800: '#8B1245',
-        900: '#411526',
-      },
-      white: '#FFFFFF',
-      transparent: 'transparent',
-      shadow: '#101828',
-    },
-    fontSize: {
-      xs: ['0.75rem', '1rem'],
-      sm: ['0.875rem', '1.25rem'],
-      base: ['1rem', '1.5rem'],
-      lg: ['1.125rem', '1.75rem'],
-      xl: ['1.25rem', '1.875rem'],
-      '2xl': ['1.5rem', '2rem'],
-      '3xl': ['1.875rem', '2.25rem'],
-      '4xl': ['2.25rem', '2.5rem'],
-      '5xl': ['3rem', '3rem'],
-      '6xl': ['3.75rem', '3.75rem'],
-      '7xl': ['4.5rem', '4.5rem'],
-    },
-    fontWeight: {
-      regular: '400',
-      medium: '500',
-      semibold: '600',
-      bold: '700',
-    },
-    fontFamily: {
-      'open-sans': ['var(--font-open-sans)'],
-      'ibm-plex-mono': ['var(--font-ibm-plex-mono)'],
-    },
-    extend: {
-      screens: { xs: '670px' },
-      backgroundImage: {
-        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-        'gradient-subtle':
-          'linear-gradient(180deg, theme(colors.neutral.100 / 50%) 0%, theme(colors.neutral.100 / 0%) 48.32%)',
-        'gradient-subtle-dark':
-          'linear-gradient(180deg, theme(colors.neutral.900 / 50%) 0%, theme(colors.neutral.900 / 0%) 48.32%)',
-        'gradient-subtle-gray':
-          'linear-gradient(180deg, theme(colors.neutral.900) 0%, theme(colors.neutral.900 / 80%) 100%)',
-        'gradient-subtle-white':
-          'linear-gradient(180deg, theme(colors.white) 0%, theme(colors.white / 80%) 100%)',
-        'gradient-glow-backdrop':
-          'radial-gradient(8em circle at calc(50%) 10px, theme(colors.green.500), transparent 30%)',
-      },
-      boxShadow: {
-        xs: '0px 1px 2px 0px theme(colors.shadow / 5%)',
-        lg: '0px 4px 6px -2px theme(colors.shadow / 3%), 0px 12px 16px -4px theme(colors.shadow / 8%)',
-      },
-      spacing: { '4.5': '1.125rem', '18': '4.5rem' },
-      aria: { current: 'current="page"' },
-      maxWidth: { '8xl': '95rem' },
-      animation: {
-        surf: 'surf 1s infinite ease-in-out',
-      },
-      keyframes: {
-        surf: {
-          '0%': { transform: 'translate(0, 0)' },
-          '25%': { transform: 'translate(0, 6px)' },
-          '50%': { transform: 'translate(0, -6px)' },
-          '75%': { transform: 'translate(0, 3px)' },
-          '100%': { transform: 'translate(0, 0)' },
-        },
-      },
-    },
-  },
-  darkMode: ['selector', '[data-theme="dark"]'],
-  plugins: [
-    require('@savvywombat/tailwindcss-grid-areas'),
-    require('@tailwindcss/container-queries'),
-  ],
-} satisfies Config;
+export { default } from '../../tailwind.config';
```

Comment:
IMO tailwind config should be part of the UI components package as a base tailwind config and we should be able to import it on the Website package.

Reply:
This also means that font loading should be done within the UI components package instead on Next.js-side.

Meaning, we should stop using `next-fonts` and use something that is pure react-ish and that can integrate natively with Tailwind.

Reply:
Meaning this package (ui-components) should also export a base CSS file that is something like:

```css
/* global.css */
@tailwind base;
@tailwind components;
@tailwind utilities;

/* update this to use the two fonts we use */
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap");
```

Reply:
Wow, I'm not a fan of what you're proposing. Why does `next-font` exist, it's to prevent the user of the page from depending on google. but if we do what you suggest that's what's going to happen. Our user will have to go through google. We would therefore need to add a file download system to the building time. To note that `googleapis` is blocked by not badly of proxy http in company or other thus this user would not have the font.

Reply:
>  it should be it's own discussion, and not part of this PR (IMO).

So I think we need to find a solution for this that doesn't depend on Google and then look for the most refined solution.

Reply:
It is important to mention that all that next-fonts does is download the font during build-time and bundle it within our assets. It still imports the font the same way with a `@import url`

Reply:
There are a couple of issues with next-fonts and why it doesn't work for us:

- We cannot use `net-fonts` in the UI components package. That package is agnostic and should only require TailwindCSS and React.
  - The fonts are essential for the UI components; without the right font families, our Storybooks and Visual Regression tests will have no meaning.
- We shouldn't rely on Next.js to have custom fonts within our Application. `next-fonts` is just an easy one-liner abstraction that is nice for most simple projects, but not here.

>  Our user will have to go through google. 

Our servers will anyway need to go through Google, and Google Fonts is also used everywhere across the internet. The chances that these fonts will already be cached by the time they need to request them from our website (at least Open Sans) reduce already the extra request they need to make to our infrastructure to load the font.

> Why does next-font exist, it's to prevent the user of the page from

Just because something exists doesn't mean we need to use it.

Reply:
I actually did mean for everything. There's no point of having next-fonts if the ui-components package already loads the fonts 🤷

Reply:
Otherwise we are double loading fonts.